frontend/src/Components/Form/FormInputButton.tsx

```diff
@@ -0,0 +1,45 @@
+import classNames from 'classnames';
+import React from 'react';
+import Button from 'Components/Link/Button';
+import { LinkProps } from 'Components/Link/Link';
+import SpinnerButton from 'Components/Link/SpinnerButton';
+import { kinds } from 'Helpers/Props';
+import styles from './FormInputButton.css';
+
+export interface FormInputButtonProps extends Omit<LinkProps, 'children'> {
+  children: Required<LinkProps['children']>;
+  canSpin?: boolean;
+  isLastButton?: boolean;
+}
+
+function FormInputButton({
+  className = styles.button,
+  children,
```

Comment:
Can pass this through via `otherProps`

Reply:
It's not because it was passed in via `otherProps`, it's because `children` was defined as `children: PropTypes.element`, `ReactNode` (which is inherited from `ComponentPropsWithoutRef` allows for `string`